tests/Doctrine/Tests/Common/Reflection/ExampleAnnotationClass.php

```diff
@@ -11,4 +11,11 @@
  */
 class ExampleAnnotationClass {
 
+  const foo = \stdClass::class;
+
+  /**
+   * Example with comment.
+   */
+  const bar = \stdClass::class;
```

Comment:
@Sam152 these additional symbols should probably be in a separate class/test - try not modifying existing test assets, please